index.html

```diff
@@ -101,6 +86,38 @@ <h4 class="about-me-title">A Little Bit About Me...</h4>
             </div>
         </div>
     </main>
-    <!--<main id="contact"></main>-->
+    <main id="contact">
+        <h4 class="title brand">Get in Contact</h4>
+        <div class="contact-container">
+            <div class="contact-area">
+                <a href="https://github.com/HazDuck" target="_blank"><img class="contact-pic" src="img/github-blue.svg"></a>
```

Comment:
images don't need thier own class here I reckon, target them with '.contact-container img'

Reply:
Repeat this advice for each contact!